### ISSUE TITLE
Toolchain: ELPA download fails

### ISSUE THREAD

**Issue by OndrejMarsalek** [cp2k/cp2k#442]
The download of ELPA in the toolchain fails, at least on my Ubuntu 18.04.2. The reason is certificate verification:

```
$ wget https://elpa.mpcdf.mpg.de/html/Releases/2017.05.003/elpa-2017.05.003.tar.gz
--2019-06-26 17:18:49--  https://elpa.mpcdf.mpg.de/html/Releases/2017.05.003/elpa-2017.05.003.tar.gz
Resolving elpa.mpcdf.mpg.de (elpa.mpcdf.mpg.de)... 130.183.17.20
Connecting to elpa.mpcdf.mpg.de (elpa.mpcdf.mpg.de)|130.183.17.20|:443... connected.
ERROR: cannot verify elpa.mpcdf.mpg.de's certificate, issued by ‘CN=MPG CA - G02,O=Max-Planck-Gesellschaft,L=Muenchen,ST=Bayern,C=DE’:
  Unable to locally verify the issuer's authority.
To connect to elpa.mpcdf.mpg.de insecurely, use `--no-check-certificate'.
```

As per the error message, adding `--no-check-certificate` helps. Could that be added to the toolchain at least as an option, given that checksums are calculated anyway?

**Comment by dev-zero** [comment]
Transport security is not the same as data integrity, therefore rather not.

**Comment by oschuett** [comment]
You can replace the url in `install_elpa.sh` with our [mirror](https://www.cp2k.org/static/downloads/elpa-2017.05.003.tar.gz) as part of your #428.

**Comment by oschuett** [comment]
Fix via 2c11076efce1cf7731b388d7968aae466770e57b.
